multi-agent/tests/integration/blueprints/test_workflow_editing_scenarios.py

```
@@ -0,0 +1,883 @@
+        We cannot emulate true parallelism with mongomock, so we simulate the
+        race by patching ``update_with_version`` to unconditionally return
+        ``None`` (as it does on a real OCC conflict).  This tests the service
+        layer's response without depending on DB internals.
+        """
+        bid = service.create_draft(identity=alice, draft_dict=_make_spec("V1"))
+
+        # Patch the OCC write to simulate a concurrent modification conflict.
+        monkeypatch.setattr(
+            service._repo,
+            "update_with_version",
+            lambda blueprint_id, spec, rid_refs, expected_version: None,
```

Ruff (0.15.17)

[warning] Unused lambda argument: `blueprint_id`

(ARG005)